framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -321,6 +328,24 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 			{ height: newBlockHeader.height },
 		);
 
+		const events = await this._sidechainAPIClient.invoke<JSONObject<chain.EventAttr[]>>(
+			'chain_getEvents',
+			{ height: newBlockHeader.height },
+		);
+		if (events && events.length > 0) {
+			const ccmSendSuccessEvents = events.filter(
+				eventAttr =>
+					eventAttr.name === CCM_SEND_SUCCESS && eventAttr.module === MODULE_NAME_INTEROPERABILITY,
```

Comment:
you also need to filter ccmProcessed based on forwarded=1 , also you need to find schema for the event data to decode. event data is i think in the serialized format.

Reply:
I have improved this part in the issue https://github.com/LiskHQ/lisk-sdk/issues/7890 which I'm working on so we can ignore this part here and solve it in my PR